lib/litmus/type/string.ex

```diff
@@ -155,6 +168,19 @@ defmodule Litmus.Type.String do
     {:ok, params}
   end
 
+  @spec included(t, term, map) :: {:ok, map} | {:error, String.t()}
+  defp included(%__MODULE__{included: nil}, _field, params) do
+    {:ok, params}
+  end
+
+  defp included(%__MODULE__{included: included}, field, params) do
+    if Map.get(params, field) in included do
+      {:ok, params}
+    else
+      {:error, "#{field} isn't into the list."}
```

Comment:
I think it might be helpful to return the valid values in the error to provide a more helpful message. Taking inspiration from [Joi](https://github.com/hapijs/joi) we could do something like this:

```elixir
defp included(%__MODULE__{valid: valid}, field, params) do
  if Map.get(params, field) in included do
    {:ok, params}
  else
    {:error, "#{field} must be one of #{valid}"}
  end
end
```